guile/guile-gnutls/Portfile

```diff
@@ -13,13 +14,18 @@ checksums           rmd160  4ac751aca5616735001f630951fdd0942d9a73c4 \
                     sha256  5b4cb926032076ec346bb5c0bc0d0231f968fe0f565913cc16934bb793afb239 \
                     size    747952
 
+gitlab.setup        gnutls guile ${version} v
+
 description         Guile bindings for GnuTLS
 long_description    Guile-GnuTLS provides Guile bindings for the GnuTLS library.
 
 distname            ${name}-${version}
 homepage            http://www.gnutls.org/
 master_sites        https://gitlab.com/-/project/40217954/uploads/9060bc55069cedb40ab46cea49b439c0/
 
+extract.suffix      .tar.gz
+use_bzip2           no
```

Comment:
that could perhaps be because you're overriding the `master_sites`; it would be strange if the default in the `gitlab` PortGroup would be incorrect. Unless upstream is doing something unusual of course...

Reply:
ah okay, now I remember the gitlab PG doesn't provide the option to get a released version. 
Sorry, I missed that and clearly didn't look closely enough - please leave as you had it!
L